### PR TITLE
fix (ContextualMenu): ContextualMenu list items now have correct hover state in HCM

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Previous Behavior

- When a `ContextualMenu` item is hovered in High Contrast Mode, there is no hover state style:

	- image

## New Behavior

- When a `ContextualMenu` item is hovered in High Contrast Mode, there is now a hover state style:

	- image

## Related Issue(s)

* Fixes ##25330

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/014041b7447b02856ae50638de71bdb829b0a759'>014041b7447b02856ae50638de71bdb829b0a759</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-c96185f0-24c9-47f5-b72b-1c11b3520979.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-c96185f0-24c9-47f5-b72b-1c11b3520979.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26440&commitId=d715856f058fb06f9c493c7aec98196e91022825&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1043 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Root.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Root.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/BaseButton-mount_base.html">1206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/BaseButton-mount.html">1192</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Breadcrumb-mount_base.html">2849</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Breadcrumb-mount.html">2809</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Checkbox-mount_base.html">2641</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Checkbox-mount.html">2656</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/CheckboxBase-mount_base.html">2370</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/CheckboxBase-mount.html">2378</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ChoiceGroup-mount_base.html">4265</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ChoiceGroup-mount.html">4281</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ComboBox-mount_base.html">1152</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ComboBox-mount.html">1171</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/CommandBar-mount_base.html">9245</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/CommandBar-mount.html">9319</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ContextualMenu-mount_base.html">9966</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ContextualMenu-mount.html">10008</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DefaultButton-mount_base.html">1384</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DefaultButton-mount.html">1357</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRow-mount_base.html">3353</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRow-mount.html">3340</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRowFast-mount_base.html">3376</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRowFast-mount.html">3375</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRowNoStyles-mount_base.html">3241</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DetailsRowNoStyles-mount.html">3211</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Dialog-mount_base.html">2934</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Dialog-mount.html">2927</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DocumentCardTitle-mount_base.html">573</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/DocumentCardTitle-mount.html">572</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Dropdown-mount_base.html">3131</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Dropdown-mount.html">3131</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/FocusTrapZone-mount_base.html">1955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/FocusTrapZone-mount.html">1942</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/FocusZone-mount_base.html">1918</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/FocusZone-mount.html">1957</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-mount_base.html">1809</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-mount.html">2032</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-virtual-rerender_base.html">1103</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-virtual-rerender.html">1093</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1598</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedList-virtual-rerender-with-unmount.html">1581</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-mount_base.html">560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-mount.html">567</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-virtual-rerender_base.html">543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-virtual-rerender.html">531</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">560</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">559</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/IconButton-mount_base.html">1775</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/IconButton-mount.html">1918</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Label-mount_base.html">746</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Label-mount.html">750</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Layer-mount_base.html">4180</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Layer-mount.html">4209</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Link-mount_base.html">855</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Link-mount.html">850</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/MenuButton-mount_base.html">1604</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/MenuButton-mount.html">1612</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/MessageBar-mount_base.html">2312</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/MessageBar-mount.html">2307</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Nav-mount_base.html">3080</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Nav-mount.html">3074</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/OverflowSet-mount_base.html">1421</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/OverflowSet-mount.html">1389</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Panel-mount_base.html">2493</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Panel-mount.html">2507</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Persona-mount_base.html">1295</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Persona-mount.html">1279</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Pivot-mount_base.html">1497</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Pivot-mount.html">1529</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/PrimaryButton-mount_base.html">1492</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/PrimaryButton-mount.html">1502</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Rating-mount_base.html">6953</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Rating-mount.html">6922</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SearchBox-mount_base.html">1475</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SearchBox-mount.html">1506</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Shimmer-mount_base.html">2935</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Shimmer-mount.html">2893</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Slider-mount_base.html">2085</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Slider-mount.html">2088</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SpinButton-mount_base.html">4215</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SpinButton-mount.html">4274</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Spinner-mount_base.html">822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Spinner-mount.html">822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SplitButton-mount_base.html">2794</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SplitButton-mount.html">2847</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Stack-mount_base.html">865</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Stack-mount.html">868</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/StackWithIntrinsicChildren-mount_base.html">2332</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/StackWithIntrinsicChildren-mount.html">2228</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/StackWithTextChildren-mount_base.html">4963</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/StackWithTextChildren-mount.html">4930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SwatchColorPicker-mount_base.html">9416</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/SwatchColorPicker-mount.html">9421</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TagPicker-mount_base.html">2442</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TagPicker-mount.html">2300</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TeachingBubble-mount_base.html">73288</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TeachingBubble-mount.html">73686</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Text-mount_base.html">819</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Text-mount.html">830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TextField-mount_base.html">1518</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/TextField-mount.html">1546</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-mount_base.html">1452</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-mount.html">1437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-virtual-rerender_base.html">1132</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-virtual-rerender.html">1136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2002</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1992</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Toggle-mount_base.html">1138</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/Toggle-mount.html">1108</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/buttonNative-mount_base.html">527</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26440/perf-test/buttonNative-mount.html">527</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-c96185f0-24c9-47f5-b72b-1c11b3520979.json (hunk 1 above)
Comment (TristanWatanabe) [reply change/@fluentui-react-c96185f0-24c9-47f5-b72b-1c11b3520979.json:1]: Flaky tests

**Pull request closed without merging**